README.md

```diff
@@ -141,66 +148,51 @@ def apply(
 import zio.sqs.{SqsStream, SqsStreamSettings}
 
 SqsStream(
-  client,
   queueUrl,
   SqsStreamSettings(stopWhenQueueEmpty = true, waitTimeSeconds = Some(3))
 ).foreach(msg => UIO(println(msg.body)))
 ```
 
-### Helpers
-
-The `zio.sqs.Utils` object provides a couple helpful functions to create a queue and find a queue URL from its name.
-
-```scala
-def createQueue(
-  client: SqsAsyncClient,
-  name: String,
-  attributes: Map[QueueAttributeName, String] = Map()
-): Task[Unit]
-
-def getQueueUrl(
-  client: SqsAsyncClient,
-  name: String
-): Task[String]
-```
-
 ### Full example
 
 ```scala
+import io.github.vigoo.zioaws
+import io.github.vigoo.zioaws.sqs.Sqs
 import software.amazon.awssdk.auth.credentials.{ AwsBasicCredentials, StaticCredentialsProvider }
 import software.amazon.awssdk.regions.Region
-import software.amazon.awssdk.services.sqs.SqsAsyncClient
+import zio.clock.Clock
 import zio.sqs.producer.{ Producer, ProducerEvent }
 import zio.sqs.serialization.Serializer
 import zio.sqs.{ SqsStream, SqsStreamSettings, Utils }
-import zio.{ App, IO, Task, UIO, ZEnv, ZIO }
-
-object TestApp extends App {
-
-  override def run(args: List[String]): ZIO[ZEnv, Nothing, Int] =
-    (for {
-      client <- Task {
-                 SqsAsyncClient
-                   .builder()
-                   .region(Region.of("ap-northeast-2"))
-                   .credentialsProvider(
-                     StaticCredentialsProvider
-                       .create(AwsBasicCredentials.create("key", "key"))
-                   )
-                   .build()
-               }
-      queueName = "TestQueue"
-      _         <- Utils.createQueue(client, queueName)
-      queueUrl  <- Utils.getQueueUrl(client, queueName)
-      producer  = Producer.make(client, queueUrl, Serializer.serializeString)
-      _ <- producer.use { p =>
-            p.produce(ProducerEvent("hello"))
-          }
-      _ <- SqsStream(
-            client,
-            queueUrl,
-            SqsStreamSettings(stopWhenQueueEmpty = true, waitTimeSeconds = Some(3))
-          ).foreach(msg => UIO(println(msg.body)))
-    } yield 0).foldM(e => UIO(println(e.toString)).as(1), IO.succeed)
+import zio.{ ExitCode, UIO, URIO, ZIO, ZLayer }
+
+object TestApp extends zio.App {
+  val queueName = "TestQueue"
+
+  val client: ZLayer[Any, Throwable, Sqs] = zioaws.netty.default >>>
+    zioaws.core.config.default >>>
+    zioaws.sqs.customized(builder =>
+      builder
+        .region(Region.of("ap-northeast-2"))
```

Comment:
just wondering: there is no zio-aws wrapped for this builder, right?

Reply:
The SQS client builder does not have any SQS-specific options so instead of this the configuration could come from the `AwsConfig` layer. It has a `zio-config` based layer definition, with that it could be:

```scala
ZLayer.success(CommonAwsConfig(region = Some(Region.of("ap-northeast-2")), credentialsProvider = ..., endpointOverride = None, commonClientConfig = None)) >>> zioaws.core.config.configured() >>> zioaws.sqs.live
```

Reply:
The idea of the `AwsConfig` layer is that it:
- gets the http backend as a dependency and applies it to the AWS client builder
- provides all the common settings for the client builder

There are a few AWS clients that has some custom options too, and that's not desribed by the schema. Those custom properties currently can only be set with the `servicelayer.customized(builderFn)` way as seen above. I was thinking of reverse engineering them from the code generator with reflection but there are so few of them that I think it does not worth it.